package.json

```diff
@@ -8,6 +8,10 @@
   "license": "UNLICENSED",
   "main": "index.js",
   "typings": "index.d.ts",
+  "files": [
+      "/include/*.js",
+      "/src/**/*.js"
```

Comment:
Needs the following:

* index.js
* index.d.ts

Reply:
Thanks! Added the two files in c86230c.

NOTE: `index.js` is automatically included as it's specified as `main`, but I added it to `files` anyway just to be more explicit.